tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -841,6 +858,19 @@ impl Core {
 
         park.shutdown(&handle.driver);
     }
+
+    fn tune_global_queue_interval(&mut self, worker: &Worker) {
+        let next = self
+            .stats
+            .tuned_global_queue_interval(&worker.handle.shared.config);
+
+        // Smooth out jitter
+        //
+        // `u32::abs_diff` is not available on Tokio's MSRV.
```

Comment:
nit, not a big deal: i might have put this comment on the abs_diff function definition, instead?